src/services/configuration/config/config.service.ts

```diff
@@ -28,7 +28,7 @@ export class KonfigService {
       {
         name: 'Azure Active Directory',
         label: 'Log in with Azure AD',
-        icon: '',
+        icon: '/images/azure.svg',
```

Comment:
this should be a full URL, not relative - it can be an image we have pinned on ipfs for example